content/docs/user-guide/assets/pipeline/asset-dependencies-and-identifiers.md

```diff
@@ -13,6 +13,22 @@ Both source and product assets might have dependencies. Source asset and job dep
 Source dependencies are a process time concept. This means that this dependency type is used when processing assets for your project, source dependencies are not used at runtime.
 
 A source dependency triggers **Asset Processor** to rerun the job process step whenever the file declared as a source dependency changes. For example, image files declare a source dependency on the `.preset` file containing the preset information used to process the image, so if the preset changes the image will be reprocessed and use the new preset settings.
+
+### Declaring source dependencies
+
+In the Job Creation step of asset processing, a JobDescriptor is created to describe the work to be done to process a source asset.
+
+One of the fields on the CreateJobsResponse is m_sourceFileDependencyList, a vector of SourceFileDependency objects. A SourceFileDependency should be populated with enough information for the Asset Processor to track this dependency, so that it can re-run this job if the declared dependency is changed.
+
+A SourceFileDependency can be created with a path, or the UUID of a source asset. When possible, using a UUID is better because it will be stable in more circumstances.
+
+Source dependencies do not have to be on other source assets, and can be on other source files. In these cases, a UUID will not be available, because it's not generated for non-asset files.'
+
+If a path must be used, it can be an absolute path or relative, and can make use of wildcards. If a path needs to be used instead of a UUID, absolute paths can be more stable than relative paths here because this data is specific to the local machine, so pathing differences across machines won't matter. Relative paths cannot include directory change markers across root drives, so if a source asset is in a scan folder on one drive root (such as C:/Projects/MyGame/Assets) and a source dependency is in a folder on another drive root (such as E:/Gems/SomeGem/Assets), a relative path will not be usable.
+
+If the source dependency is a source asset, it must be in a scan directory. Read more about [scan directories here.](/docs/user-guide/assets/pipeline/scan-directories/)
+
+If the source dependency is not a source asset and is instead a non-asset file, it can be in other locations outside of scan directories.
```

Comment:
big suggestion here, but it's just adding code tags per the style guide.

```suggestion
In the Job Creation step of asset processing, a `JobDescriptor` is created to describe the work to be done to process a source asset.

One of the fields on the `CreateJobsResponse` is `m_sourceFileDependencyList`, a vector of `SourceFileDependency` objects. A `SourceFileDependency` should be populated with enough information for the Asset Processor to track this dependency, so that it can re-run this job if the declared dependency is changed.

A `SourceFileDependency` can be created with a path, or the UUID of a source asset. When possible, using a UUID is better because it will be stable in more circumstances.

Source dependencies do not have to be on other source assets, and can be on other source files. In these cases, a UUID will not be available, because it's not generated for non-asset files.'

If a path must be used, it can be an absolute path or relative, and can make use of wildcards. If a path needs to be used instead of a UUID, absolute paths can be more stable than relative paths here because this data is specific to the local machine, so pathing differences across machines won't matter. Relative paths cannot include directory change markers across root drives, so if a source asset is in a scan folder on one drive root (such as `C:/Projects/MyGame/Assets`) and a source dependency is in a folder on another drive root (such as `E:/Gems/SomeGem/Assets`), a relative path will not be usable.

If the source dependency is a source asset, it must be in a scan directory. Read more about [scan directories here.](/docs/user-guide/assets/pipeline/scan-directories/)

If the source dependency is not a source asset and is instead a non-asset file, it can be in other locations outside of scan directories.
```